lib/users.ts

```diff
@@ -96,12 +101,58 @@ export const getOrCreateUser = async ({
           name: true,
           email: true,
           privileges: true,
+          active: true,
```

Comment:
The active propery value only seems to be returned for a user who exists but did not have base privileges associated.  A few things we might want to do which this fn.  Update the return type to include `active: boolean` and update the other prisma calls to include `active` in the selection set.